packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -18,8 +18,9 @@ $usa-banner-font-family: $font-sans;
   flex-direction: row;
   font-size: 12px;
   font-weight: $font-normal;
+  line-height: 1.2;
```

Comment:
Just curious, Why the line height change? I couldnt see a big difference with these style changes

Reply:
Mainly changed it because uswds has line-height of around 1.1 ~ 1.2 and the difference is more noticeable when I zoom .